analytics.proto

```diff
@@ -31,4 +31,14 @@ message Analytics {
 
   optional uint32 postgresql_version = 5;
 
+  message RBACStats {
+    optional uint32 users = 1;
+    optional uint32 groups = 2;
+    optional uint32 domains = 3;
```

Comment:
I'd suggest moving this to just be a peer of rbac_stats (outside of this `message`). That makes more sense to me, but if you decline I'm ok with that. You decide.